fedora_messaging/tests/unit/test_cli.py

```diff
@@ -130,8 +128,8 @@ def test_good_cli_bindings(self, mock_consume, mock_importlib):
             ],
             queues={
                 "qn": {
-                    "durable": True,
-                    "auto_delete": False,
+                    "durable": False,
```

Comment:
Do you think it's better to have `durable` set to `False` by default? Why is that?

Reply:
I was on the fence here. The reason this changed was that the configuration defaults were non-durable and auto-deleted, but the CLI defaults were durable and not auto-deleted. I can see advantages for both, but it would be good if it matched whatever the configuration default is so using the API and the CLI behave the same.

*Technically* this is a backwards-incompatible change so we should probably bump the version to 2.

Reply:
I totally agree with the need for coherence. I don't see the advantage of non-durable, how is it useful?

Reply:
Oh I see. Wouldn't "durable & auto-delete" fix this issue? The queue will survive a broker restart (which may happen independently from the client restart) but will be deleted when the client quits? Ah, but if you don't set a queue name and the broker restarts while the client is not restarted, it will reconnect and generate a different queue name, no? Or will it re-use the existing one?

I think setting auto-restart to false is useful when you want messages to keep landing in your queue when you restart a client, but that's more appropriate for well-established clients that we setup in infra, so we can set that in the config file.

Ah, I don't know. Keeping the current defaults has the slight advantage of allowing us to keep the major version number. Which reminds me I haven't set all options in the infra config files, and I should if we want to change those defaults one day.

Reply:
> I think setting auto-restart to false is useful when you want messages to keep landing in your queue when you restart a client, but that's more appropriate for well-established clients that we setup in infra, so we can set that in the config file.

It generates a random UUID each time the process starts. I see the defaults as being things you don't use, probably not even in a development environment. It's mostly for "I just installed it and want to see what happens when I do <x>". 

> Ah, I don't know. Keeping the current defaults has the slight advantage of allowing us to keep the major version number. Which reminds me I haven't set all options in the infra config files, and I should if we want to change those defaults one day.

Haha, yeah, I kept going back and forth on it and I don't have a good answer. I'm not too concerned about bumping the major version number since it's a super tiny change, it's more I don't know what the best default is. I lean towards the "durable=False, auto-delete=True" because it's tidiest, but "durable=False, auto-delete=False" will clean up on broker restarts so maybe that's the nicest balance. "durable=True, auto-delete=False" is what people will want in production, so an argument could be made that that's what the defaults should be so users don't trip up on that.

Reply:
And coïncidentally, that's what our current defaults are! :-) Such good planning.

Reply:
I've thought about it a bit and I think I'd like to defer changing the queue settings to a different PR. Unfortunately, when changing the tests I noticed they have some ordering issues which I'm going to fix in a separate PR as well, then this PR will be ready.